general/1_terminal/Oleg-Levitskiy/task_4.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cat zoola_chat_result.json | jq .messages[].text
```

Comment:
I'll accept this answer.

If you'd like an additional assignment, make it so the messages with links and mentions look the same as the messages without them. Currently they are rendered as arrays.